### ISSUE TITLE
4th Learning

### ISSUE THREAD

**Issue by roqhdehd502** [roqhdehd502/Franklin-Detecter#5]
### 진행 전
- 손실율(Loss)의 진행도가 미비하여 두 가지 경우로 학습 테이블 세팅
1.  배치 사이즈는 그대로 학습률을 높임
2.  배치 사이즈를 낮추고 학습률은 그대로
- 기존 신경망은 현행 유지
###  진행 중
- Tensorflow, numpy, matplotlib Import
- 이미지 파일을 불러온 후 텍스트 파일과 통해 이미지 전처리
- CNN 구현
![NaminNeuralNetwork_mk2](https://user-images.githubusercontent.com/47406388/69316168-95aee800-0c7b-11ea-9614-5ce1791d6ce8.jpg)
- Softmax로 확률 구현
- Adam으로 Optimizer 구현
- 배치사이즈와 학습률을 두 가지 경우로 개별 학습
- 훈련 파일을 ckpt 확장자로 저장 후 시험
###  진행 후
-  배치 사이즈 그대로, 학습률을 높인 경우
[4th Learning.txt](https://github.com/roqhdehd502/Franklin-Detecter/files/3873322/4th.Learning.txt)
- 배치 사이즈 낮춤, 학습률을 그대로 한 경우
[5th Learning.txt](https://github.com/roqhdehd502/Franklin-Detecter/files/3873325/5th.Learning.txt)
-  전자의 경우 학습률을 높일 때 손실율의 차이는 있었으나 기대한 만큼의 결과가 나오지 않았음
-  후자의 경우 학습률이 진자 운동 하듯 기대한 만큼의 결과가 어느정도 나왔음
-  그러나 둘 모두 테스트 파일에 적용결과 정확도가 기존과 차이가 없음(약 33%)
- 신경망의 커널을 조정 혹은 버그인지 조사가 필요함
